src/components/Calculator.jsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-nested-ternary */
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/no-unused-state */
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] It is not good programming practice to disable linter errors, kindly remove these diable linter comments and fix the errors as this will make your codebase cleaner.

Reply:
Hallo Munsa1, thank you for the review, but kindly note that linters trigger errors when using class components instead of functional ones. Also when giving classNames dynamically to the button element I had to use nested ternary operators which is a linter error too so I had to disable it.

Reply:
Hi @ndikumanaisaie in the next milestone you would use function based components so there would be no need to diable the linters 👍🏿